### PR TITLE
HPCC-24173 Remove doc references to Python2

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: g-pan <greg.panagiotatos@lexisnexis.com>
Note some references to Python2 due to publishing the contents of environment.conf.  References to python2 remain in environment.conf.  

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Successful integration testing: http://10.224.20.18/view/Docs-gp/job/DocBuild-01-GPRepo/

### CODE REVIEW

File: docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 1]
```diff
@@ -1258,34 +1258,35 @@ mpStart=7101
...
 
-# Although HPCC platform includes plugins for both Python2 and Python3, 
-# only one may be safely enabled at a time
-# as the Python libraries export the same symbols for both versions. 
-# Enabling both may lead to unpredicatable results
+# Python plugins can call python cleanup code on exit, but this seems to cause lockups in some Tensorflow examples
+# In most cases, skipping the cleanup is harmless and avoids these lockups
+skipPythonCleanup=true
+
+# Although HPCC platform includes plugins for both Python2 and Python3, only one may be safely enabled at a time
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:1281]: This paragraph mentions Python 2 support

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 2]
```diff
@@ -2434,20 +2434,10 @@ add1(10);
...
-        <para>By default the Python2 plugin is present but disabled, and the
-        Python3 plugin is present and enabled. If you want to use Python2
-        modify your environment.conf file according to the example in the
-        file.</para>
-
-        <para>A cluster updated from an older version may not have the
-        appropriate entries in the environment.conf file to support either
-        version of Python.</para>
+        symbols for different versions. The Python3 plug-in is present and
+        enabled while Python2 is no longer supported.</para>
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml:2440]: The HPCC Systems platform supports Python3 and the Python3 plug-in is present and
        enabled. Python2 is no longer supported.</para>

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
comments inline

File: docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml (hunk 1 above)
Comment (g-pan) [reply docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:1281]: That is correct. This is depicting contents the environment.conf  
referred to in the comment above.

File: docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 3]
```diff
@@ -1282,7 +1282,7 @@ skipPythonCleanup=true
 # as the Python libraries export the same symbols for both versions. Enabling both may lead to unpredicatable results
 # including segfaults or undefined symbol errors.
 #
-# If you would prefer to use python 2 and disable python3, change the line below to read
+# If you would prefer to use python2 and disable python3, change the line below to read
 #  additionalPlugins=python2
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:1286]: Still references python2

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]

**Review by JamesDeFabia — APPROVED** [review]
Good from my POV

**Comment by g-pan** [comment]
good to merge

**Comment by g-pan** [comment]
@ghalliday this is good to merge now

**Review by ghalliday — APPROVED** [review]

**Pull request merged**